feature-detects/css/remunit.js

```diff
@@ -25,7 +25,7 @@ define(['Modernizr', 'createElement'], function( Modernizr, createElement ) {
     try {
       div.style.fontSize = '3rem';
     }
-    catch( er ) {}
+    catch ( er ) {}
```

Comment:
Can we add a rule here to make it drop the whitespace inside the parenthesis too?